modules/ROOT/pages/architecture.adoc

```diff
@@ -52,7 +52,7 @@ This diagram shows the TCP load balancer used to load balance requests across th
 
 For internal load balancers:
 
-* *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+* *Shared Mode* is the default setting if no dedicated internal load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
```

Comment:
I'm not sure if this change is necessary, it's just that the other way around it sounds odd to me, please ignore the suggestion if you think the original text is ok. 
```suggestion
* *Shared Mode* is the default setting if no dedicated internal load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the specified amount of CPU and memory.
```